examples/14-hierarchical_pipeline.ipynb

```diff
@@ -4,37 +4,36 @@
    "cell_type": "markdown",
```

Comment:
<p>Why did the metrics change?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/tinkoff-ai/etna/pull/1330/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='80'/>